### ISSUE TITLE
Web: lists not accessible with Voice Over

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#94404]
Mac, Latest Chrome, open VoiceOver start VS Code web 

1. Focus on any list, Voice Over does not read any item 🐛 
Editor and other parts of the UI are nicely accessible.

This reproduces in the Explorer, Extensions, Quick Pick. Basically making all our web UI not accessible.
I can also try with NVDA and Orca on linux.

**Comment by isidorn** [comment]
To note: all accessibility flags under chrome://accessibility/ are checked on for me
fyi @robloures

**Comment by bpasero** [comment]
Can we ask someone for help outside of VSCode team? Setting to "On Deck" just so that it disappears from my "to triage" query...

**Comment by isidorn** [comment]
I can set it to April to look into if it works for other screen readers.
If it only does not work on VoiceOver then we can contact the VoiceOver owner.

**Comment by isidorn** [comment]
Ok, I can reproduce this on MacOS and on Linux. I can also reproduce it with Chrome and Firefox.
So it seems like something is broken on our end.  I will experiment with changing the role `application` and the list role just to check if that makes a difference.

@bpasero do we have docs in the open on how to run VS Code in the web, or just an easy way to do it? Asking because I migth ask for help from the Orca screen reader owner and it would be cool if she could easily run us in the web.

**Comment by bpasero** [comment]
@isidorn https://github.com/microsoft/vscode/wiki/How-to-Contribute#run

**Comment by isidorn** [comment]
I tried changing roles, but to no success.

@joanmarie this issue is not related to Orca, but maybe you could give us a hand? If yes, you can follow: 
https://github.com/microsoft/vscode/wiki/How-to-Contribute#run under Web and get VS Code running in the browser, turn on Orca and try to navigate in the Explorer, nothign gets read out. Same underlying issue is when using Quick Open (via ctrl+E). Might you have a clue why our list are not being read out? Note that we can repro on Linux, Mac, Windows, Chrome, Firefox so it seems like something is broken on our end.
Some quick detail how our List manage focus: top list element has focus and it uses `activedescendent` to notify the screen reader which element should have focus. This is all nicely set, but does not seem to work.
fyi @marcozehe for ideas

**Comment by joanmarie** [comment]
@isidorn: I just tried Orca with Firefox and in Chrome and the items in question were presented by Orca as I arrowed up and down. 

Just to be sure we're talking about the same items, I've attached a screenshot pointing to the two lists I believe we're talking about.

![vscode-web](https://user-images.githubusercontent.com/3228171/78799820-aa6ced80-7988-11ea-991e-201c84a3c725.png)

**Comment by isidorn** [comment]
@joanmarie thanks for trying it out! Yeah, those are the two lists we are talking about.
I just updated to latest Orca 3.37.1pre 
I am using Chrome Version 81.0.4044.92 (Official Build) (64-bit) with accessibility flags
I am on Ubuntu 18
And I can still reproduce the issue.

Tomorrow I plan to investigate more...

**Comment by joaomoreno** [comment]
@isidorn Let me know if I can help.

**Comment by isidorn** [comment]
This is actually fixed! Thus closing. Maybe my role changes tackled this
Only follow up: it seems that VoiceOver on mac does not read quick pick focus https://github.com/microsoft/vscode/issues/96578
